src/src/Helpers/Misc.cpp

```diff
@@ -423,10 +423,10 @@ void HSV2RGBW(float H, float S, float I, int rgbw[4]) {
     order = RGB_ORDER;
   } else if (H < ANGLE_240_DEG) {
     H     = H - ANGLE_120_DEG;
-    order = GBR_ORDER;
+    order = BRG_ORDER;
   } else {
     H     = H - ANGLE_240_DEG;
-    order = BRG_ORDER;
+    order = GBR_ORDER;

```

Comment:
So this was the only bug? (and line 426 of course)
Why change the defined values ? Are their int values '1' and '2' somewhere else used as integer values?  (or stored in settings)

Reply:
I only changed these values to match the changed order of the rgbw values 
To use the RGB scheme for the order is actually a bit misleading in my opinion...
(huh.. now i realized i only had to change the order of the order.. nevermind. hard to explain :) ) 

> Are their int values '1' and '2' somewhere else used as integer values? (or stored in settings)

Good question... I don't think so. Probably unnecessary.